reactive/kotlinx-coroutines-reactive/src/Await.kt

```diff
@@ -158,7 +162,7 @@ private suspend fun <T> Publisher<T>.awaitOne(
                     if ((mode == Mode.SINGLE || mode == Mode.SINGLE_OR_DEFAULT) && seenValue) {
                         subscription.cancel()
                         if (cont.isActive)
```

Comment:
While we are here, please wrap this statement in curly braces

Reply:
I think adding https://github.com/Kotlin/kotlinx.coroutines/pull/2586/commits/4baba2bc2aa5fd21ca2ef287cb9bd642debeff13 to https://github.com/Kotlin/kotlinx.coroutines/pull/2586 would make for a better place for such a change.